lib/compare-file-size.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const promisify = require('bluebird').promisify;
+const glob = promisify(require('glob').glob);
+const getFileSizesPerType = require('./get-filesizes-per-type');
+
+function compareFileSize(sourceFolder, tempFolder) {
+	return glob('**/*', { nodir: true, cwd: sourceFolder })
+		.then(files => {
+			const sourceFiles = appendDirToFiles(sourceFolder, files);
```

Comment:
I'd use `const sourceFiles = files.map(file => path.join(sourceFolder, file));` The function `appendDirToFiles` doesn't instantly tell me what will happen inside. So for a oneline function I find it too much.